_posts/2023-01-25-#26347.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: pr
+date: 2023-01-25
+title: "wallet: ensure the wallet is unlocked when needed for rescanning"
+pr: 26347
+authors: [ishaanam]
+components: ["wallet"]
+host: ishaanam
+status: upcoming
+commit:
```

Comment:
Just pushed a tag: https://github.com/bitcoin-core-review-club/bitcoin/commits/pr26347
If you decide to add more links to code/commits in the PR, I recommend using links from that repo. (This is so if the PR branch changes after the review club meeting, people reading the notes can still see what bits of code it's talking about).
```suggestion
commit: b2e2829
```